docs/references/references.md

```diff
@@ -218,6 +246,10 @@ subql-node -f . -d "https://api.subquery.network/sq/subquery/dictionary-polkadot
 
 [Read more about how a SubQuery Dictionary works](../tutorials_examples/dictionary.md).
 
+### -p, --port
+
+The port the subquery indexing service binds to, by default this is set to `3000`
```

Comment:
The port the subquery indexing service binds to. By default this is set to `3000`